src/swap/SwapReviewScreen.tsx

```diff
@@ -109,7 +104,8 @@ export function SwapReviewScreen(props: Props) {
           `Failure response fetching token swap quote. ${response.status}  ${response.statusText}`
         )
       }
-      setSwapInfo(await response.json())
+      const json: SwapInfo = await response.json()
```

Comment:
i think this type is not right? `SwapInfo` includes a `userInput` field which will not be returned from the response right?

this screen seems only to need the `unvalidatedSwapTransaction` property from the `swapInfo` state variable, so should the type of this state variable be `SwapTransaction` instead?

Reply:
I also need the `approveTransaction`. Should there be a new type with both included or should swapInfo be changed to omit `validatedSwapTransaction`?